llvm/test/CodeGen/RISCV/stack-guard-tls.ll

```diff
@@ -0,0 +1,19 @@
+; RUN: llc -mtriple=riscv64-unknown-elf < %s | \
```

Comment:
Use `llvm/utils/update_llc_test_checks.py` to generate the checks.

Reply:
Thanks. I wondered how that was supposed to work.